pjmedia/src/pjmedia/vid_stream.c

```diff
@@ -1279,7 +1279,7 @@ static pj_status_t decode_frame(pjmedia_vid_stream *stream,
                                 pjmedia_frame *frame)
 {
     pjmedia_vid_channel *channel = stream->dec;
-    pj_uint32_t last_ts = 0, frm_ts = 0;
+    pj_uint32_t last_ts = -1, frm_ts = 0;
```

Comment:
perhaps it's better to introduce a new boolean flag, e.g: `last_ts_set`, because:
- negative value does not fit unsigned type,
- RTP ts is 32 bits, so any value (including `0` & `-1`) may still match a real RTP ts.